packages/api/amplify_api/test/dart_graphql_test.dart

```diff
@@ -12,10 +12,12 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+import 'dart:async';
```

Comment:
I guess I never noticed this - but why are the other method channel tests/infra hanging around still? Outside of the configure API, we have no use for them anymore. These tests should just replace the old subscribe tests.

Reply:
I would rather do the cleanup in a single PR. There was a task we mentioned in grooming for it.